src/main/resources/static/app/request/request-review.html

```diff
@@ -1,4 +1,27 @@
 <div>
+  <h4 class="h4 h4-palga">{{'Request type'|translate}}</h4>
+  <hr>
+
+  <!-- Request type -->
+  <div class="form-group" ng-class="{ 'has-error' : registrationForm.requestType.$invalid}">
```

Comment:
`registrationForm` should be `requestEditForm`

Reply:
this fixed the issue with validation messages, thanks!